app/feature/feature-help-center/src/main/kotlin/com/hedvig/android/feature/help/center/home/HelpCenterHomeDestination.kt

```diff
@@ -149,6 +184,13 @@ private fun HelpCenterHomeScreen(
       Column(
         modifier = Modifier
           .fillMaxSize()
+          .clickable(
+            indication = null,
+            interactionSource = remember { MutableInteractionSource() },
+          ) {
+            focusManager.clearFocus() // clearing focus for search textField
+          }
```

Comment:
I think this could also be `.clearFocusOnTap()` which does the same thing internally